rust/cli/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+mod commands;
+mod utils;
+
+use crate::commands::run::{run, RunArgs};
+use crate::commands::update::update;
+use crate::commands::vacuum::{vacuum, VacuumArgs};
+use clap::{Parser, Subcommand};
+
+#[derive(Subcommand, Debug)]
+enum Command {
+    Docs,
+    Run(RunArgs),
+    Support,
+    Update,
+    Vacuum(VacuumArgs),
+}
+
+#[derive(Parser, Debug)]
+#[command(name = "chroma")]
+#[command(version = "1.0.0")]
+#[command(about = "A CLI for Chroma", long_about = None)]
+struct Cli {
+    #[command(subcommand)]
+    command: Command,
+}
+
+pub fn chroma_cli(args: Vec<String>) {
+    let cli = Cli::parse_from(args);
+
+    match cli.command {
+        Command::Docs => {
+            let url = "https://docs.trychroma.com";
+            if webbrowser::open(url).is_err() {
+                eprintln!("Error: Failed to open the browser. Visit {}.", url);
+            }
+        }
+        Command::Run(args) => {
+            run(args);
+        }
+        Command::Support => {
+            let url = "https://discord.gg/MMeYNTmh3x";
```

Comment:
nit: wonder if it's worth setting up a redirect for this like `https://trychroma.com/join-discord` since invite links can expire/be revoked